project/Dependencies.scala

```diff
@@ -115,42 +123,57 @@ object Dependencies {
   }
 
   def serverDependencies: Seq[ModuleID] = {
-    val springOrg = "org.springframework.boot"
-
-    lazy val springBootTest = springOrg % "spring-boot-starter-test" % Versions.spring
-    lazy val springBootWeb = springOrg % "spring-boot-starter-web" % Versions.spring
-    lazy val springBootConfiguration = springOrg % "spring-boot-configuration-processor" % Versions.spring
-    lazy val springBootTomcat = springOrg % "spring-boot-starter-tomcat" % Versions.spring
-    lazy val servletApi = "javax.servlet" % "javax.servlet-api" % Versions.javaxServlet
-    lazy val springFoxSwagger = "io.springfox" % "springfox-swagger2" % Versions.springfox
-    lazy val springFoxSwaggerUI = "io.springfox" % "springfox-swagger-ui" % Versions.springfox
-    lazy val springFoxBoot = "io.springfox" % "springfox-boot-starter" % Versions.springfox
-
-    // controller implicits:  java CompletableFuture -> scala Future
-    lazy val scalaJava8Compat = "org.scala-lang.modules" %% "scala-java8-compat" % Versions.scalaLangJava8Compat
-
-    // object mapper serialization
-    lazy val jacksonModuleScala = "com.fasterxml.jackson.module" %% "jackson-module-scala" % Versions.jacksonModuleScala
-
-    // Fa-db & Slick & PG dependencies
-    lazy val faDbCore = "za.co.absa.fa-db" %% "core" % Versions.fadb
-    lazy val faDbSlick = "za.co.absa.fa-db" %% "slick" % Versions.fadb
-    lazy val slickPg = "com.github.tminglei" %% "slick-pg" % Versions.slickpg
+    val zioOrg = "dev.zio"
+    val tapirOrg = "com.softwaremill.sttp.tapir"
+    val http4sOrg = "org.http4s"
+    val faDbOrg = "za.co.absa.fa-db"
+    val playOrg = "com.typesafe.play"
+
+    // zio
+    lazy val zioCore = zioOrg %% "zio" % Versions.zio
+    lazy val zioMacros = zioOrg %% "zio-macros" % Versions.zio
+    lazy val zioLogging = zioOrg %% "zio-logging" % Versions.zioLogging
+    lazy val zioConfig = zioOrg %% "zio-config" % Versions.zioConfig
+    lazy val zioConfigMagnolia = zioOrg %% "zio-config-magnolia" % Versions.zioConfig
+    lazy val zioConfigTypesafe = zioOrg %% "zio-config-typesafe" % Versions.zioConfig
+
+    // http4s
+    lazy val http4sBlazeBackend = http4sOrg %% "http4s-blaze-server" % Versions.http4sBlazeBackend
+
+    // tapir
+    lazy val tapirHttp4sZio = tapirOrg %% "tapir-http4s-server-zio" % Versions.tapir
+    lazy val tapirSwagger = tapirOrg %% "tapir-swagger-ui-bundle" % Versions.tapir
+    lazy val tapirPlayJson = tapirOrg %% "tapir-json-play" % Versions.tapir
+
+    // json
+    lazy val playJson = playOrg %% "play-json" % Versions.playJson
+
+    // Fa-db
+    lazy val faDbDoobie = faDbOrg %% "doobie" % Versions.fadb
+
+    // testing
+    lazy val zioTest = zioOrg %% "zio-test" % Versions.zio % Test
+    lazy val zioTestSbt = zioOrg %% "zio-test-sbt" % Versions.zio % Test
+    lazy val zioTestJunit = zioOrg %% "zio-test-junit" % Versions.zio % Test
+    lazy val sbtJunitInterface = "com.github.sbt" % "junit-interface" % "0.13.3" % Test
```

Comment:
this version here can be extracted to the `object Versions`

Reply:
thanks, extracted the version to Versions object